includes/controllers/pages/class-all-listings.php

```diff
@@ -39,8 +39,8 @@ public function dispatch() {
             $q->max_num_pages = 1;
         }
         wpbdp_push_query( $q );
-
-        $show_menu = isset( $this->menu ) ? $this->menu : ( ! empty ( $args['tax_query'] ) ? false : true );
+		$show_all = isset( $this->in_shortcode ) ? ! $this->in_shortcode : ( ! empty ( $args['tax_query'] ) ? false : true );
```

Comment:
This is not very readable. Can this be cleaned up a bit to make it easier to follow? Maybe we can reverse this so we can get rid of the ! and " ? false : true"